plugins/hypervisors/vmware/src/com/cloud/hypervisor/vmware/resource/VmwareResource.java

```diff
@@ -4076,8 +4059,7 @@ private void handleTargets(boolean add, List<Map<String, String>> targets, HostM
         if (targets != null && targets.size() > 0) {
```

Comment:
You can use CollectionUtils.isNotEmpty

Reply:
useful comment but does not apply to this PR, I think a PR to amend this PR would be in place here, no?